.storybook/main.js

```diff
@@ -0,0 +1,51 @@
+const path = require("path");
+const { storyTree } = require("./storyTree");
+
+const stories = ["./welcome.story.js"];
+storyTree.forEach(catList => {
+  catList.components.forEach(component => {
+    const path = `../packages/${component}/stories/${component}`;
+    stories.push(
+      `${path}.stories.mdx`,
+      `${path}.stories.js`,
+      `${path}.examples.stories.js`,
+      `${path}.backyard.stories.js`,
+      `${path}.tests.stories.js`
+    );
+  });
+});
+stories.push("./components/CodeViewer/CodeViewer.story.js");
```

Comment:
just curious why CodeViewer is outside of storyTree

Reply:
It's not a Paprika component, it's a Storybook helper component.  It's `path` does not include `packages` so the regular loop won't work.  Might be worth thinking it through a little more, but for now it's a one-off exception, so I wasn't too concerned about it.